JSDemos/Demos/Scheduler/CustomizeIndividualViews/description.md

```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, the _&laquo;Week&raquo;_ and _&laquo;Work Week&raquo;_ views of&nbsp;the Scheduler component are configured individually. The appointments are grouped by&nbsp;type (on&nbsp;the _&laquo;Week&raquo;_ view) and by&nbsp;priority (on&nbsp;the _&laquo;Work Week&raquo;_ view). In&nbsp;addition, the time window on&nbsp;the _&laquo;Work Week&raquo;_ view is&nbsp;narrowed down to&nbsp;working hours.
\ No newline at end of file
+To customize a view, configure its settings in an object of the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.
```

Comment:
An object should be an element of the **views[]** array that means this object is **in** the array, right?

```suggestion
To customize a view, configure its settings in an object in the [views[]](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views) array. You should specify the view's [type](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type) and other properties that override global Scheduler settings.
```